f2/src/FinalFeedbackPage.js

```diff
@@ -42,17 +41,14 @@ export const FinalFeedbackPage = () => {
         <Layout>
           <TrackPageViews />
           <Stack spacing={10} shouldWrapChildren>
-            <BackButton route="/thankyoupage">
-              <Trans id="finalFeedback.backButton" />
-            </BackButton>
             <H1>
               <Trans id="finalFeedback.title" />
             </H1>
             <FinalFeedbackForm
               onSubmit={data => {
                 submitToServer(data)
                 setState((state.doneFinalFeedback = true))
-                history.push('/thankYouPage')
+                history.push('/finalfeedbackthanks')
```

Comment:
Can we add a conditional here that links to finalfeedbackthanks only if state.doneforms is true? That way, we keep what we have now, but we fix the confusing scenario described in #1592

Reply:
added code so that
- if they've submitted, they go back to the thankyou page that has the report id on it
- if they have cancelled it goes to the "Thank you for helping us improve this service!" page